_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,277 @@
+
+### Properties
+
+| Name      | Type                                                                                                                                                                         | Description                                                                                                                                                                                      |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `@type`   | "CreateAttributeRequestItem"                                                                                                                                                 | Specifies the type of the Request item for parsing.                                                                                                                                              |
+| attribute | [IdentityAttribute]({% link _docs_explore/61-data-model.md %}#identityattribute) \| [RelationshipAttribute]({% link _docs_explore/61-data-model.md %}#relationshipattribute) | The IdentityAttribute or RelationshipAttribute to create for the peer within the Identity of the peer.<br>The owner of the Attribute which should be created can only be the recipient Identity. |
+
+### Response
+
+#### Item Properties
+
+| Name        | Type                                | Description                      |
+| ----------- | ----------------------------------- | -------------------------------- |
+| `@type`     | "CreateAttributeAcceptResponseItem" | The type of the ResponseItem.    |
+| attributeId | `string`                            | The id of the created Attribute. |
+
+### Combinations and usage scenarios
+
+| Attribute Type | Attribute Owner | Possible? | Automation      | Examples/Reason                                                                                                                                                                                                                                                       |
+| -------------- | --------------- | --------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          | N         |                 | Use ShareAttributeRequestItem instead.                                                                                                                                                                                                                                |
```

Comment:
Schreib in die Automation Spalte vielleicht in den Fällen, die nicht "Possible" sind, wie in der Tabelle im Confluence "N/A". Gilt natürlich genauso für alle anderen Request Items